README.md

```diff
@@ -0,0 +1,10 @@
+# clamav-files
+
+Virus definition files for clamav, downloaded from https://www.clamav.net/downloads.
+
+This repository is used when installing Archivematica using the ansible role
+https://github.com/artefactual-labs/ansible-archivematica-src .
```

Comment:
Markdown will be ok if you add a dot at the end of a URL, it will handle it successfully.